src/app/youtube/components/sorting-panel/sorting-panel.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+import { CardListService } from '../../services/card-list.service';
+import { Direction, defaultSort } from './constants';
+
+export interface ISort {
+  sortBy: string;
+  orderBy: string;
+  search: string;
+}
+
+@Component({
+  selector: 'app-sorting-panel',
+  templateUrl: './sorting-panel.component.html',
+  styleUrls: ['./sorting-panel.component.scss']
+})
+export class SortingPanelComponent {
+  @Input() isVisible: boolean;
```

Comment:
if you use @input in component so it wiuld be better to specify change detection to be OnPush